src/data/java/com/github/minecraftschurlimods/arsmagicalegacy/data/AMPatchouliBookProvider.java

```diff
@@ -300,14 +298,21 @@ protected void addBooks(Consumer<BookBuilder<?, ?, ?>> consumer) {
         shapes.build();
         components.build();
         modifiers.build();
-        TranslatedCategoryBuilder affinities = builder.addCategory("affinities", "Affinities", "Think of affinities like the elements of spell parts.$(br2)Every affinity has an essence that is used in intermediate spell crafting involving this affinity. When you use spells with a certain affinity a lot, you will shift more and more into that affinity. Your affinity depth for all affinities can be seen in the affinity tab of the $(l:blocks/occulus)Occulus$().$(br2)While you don't notice anything yet, you suspect that being too deep in one affinity might cause some things to happen in the future.", affinityHelper.getEssenceForAffinity(IAffinity.WATER));
+        TranslatedCategoryBuilder affinities = builder.addCategory("affinities", "Affinities", "", affinityHelper.getEssenceForAffinity(IAffinity.WATER))
+                .setSortnum(8);
+        affinities.addEntry("affinities", "Affinities", affinityHelper.getEssenceForAffinity(IAffinity.WATER))
+                .setPriority(true)
+                .addSimpleTextPage("Affinities are magical elements of sorts. All components (excluding some of the very powerful, reality-bending ones) have an affinity that is associated with them. That means that if you use spells with a certain affinity a lot, you will shift into that affinity.")
+                .addSimpleTextPage("Shifting into an affinity bears individual side effects, called abilities. Each affinity has different abilities. You can read about the abilities for each affinity in the dedicated chapters for them.$(br2)If you wish to see your current shift into an affinity, you can view your shifts in the Affinity tab of the $(l:blocks/occulus)Occulus$().")
+                .addSimpleTextPage("There is also an affinity essence for each affinity, which is used in intermediate crafting for spell parts associated with that affinity. Reports about lost affinity tomes have been spreading as well, though how to obtain or use them is currently subject to investigation.")
+                .build();
```

Comment:
Nope.... This won't look good 2 entries with the same item as icon also an additional entry where a category description would suffice

Reply:
Category description is too large and invasive imo; and I'm open to suggestions for a better icon.

Reply:
If you think so ...
Use a nether star for now